recipes-kernel/linux/linux-nilrt-nohz_git.bb

```diff
@@ -19,4 +19,4 @@ do_install_append() {
 
 # This is the place to overwrite the source AUTOREV from linux-nilrt.inc, if
 # the kernel recipe requires a particular ref.
-#SRCREV = ""
+SRCREV = "378d8fbcc32082ee008a0df967ebd0f41a182877"
```

Comment:
This is the hash from nilrt/22.8/5.15 but the recipe (in your other patch) points to nilrt/22.8/5.10. The hash that goes with the 5.10 branch is 9ab416c284b7f19102e9bcdf8320f3112a0450b5

Reply:
Fixed both and double-checked the other hashes again.